public/app/plugins/datasource/loki/syntax.ts

```diff
@@ -239,9 +239,19 @@ export const lokiGrammar: Grammar = {
       },
     },
   ],
+  quote: {
+    pattern: /".*?"/,
+    alias: 'string',
+    greedy: true,
```

Comment:
Looks good but I am not sure about the implications of this being booth greedy and having a lazy modifier `?` in the regex. Wouldn't it work the same if it was `/".*"/` and `greedy: false`?

Reply:
I just copied this from another syntax file :) but I think you are right, I will try.

Reply:
@aocenas I tried non greedy but it did not work. I changed the patterns to use the same pattern used for label value so that it works with escape characters as well. I tried IntelliSense for label values and work as before so it looks like adding this has not broken anything else.